src/server/plugins/wait_for_plugins_init.js

```diff
@@ -0,0 +1,28 @@
+
+const queues = new WeakMap();
```

Comment:
Maybe explain the role of `queues`? And include why it has to be a `WeakMap` (compared to just a `queue` variable). I expect it is because you're expecting multiple different `kbnServer`s?